Documentation/security/network/encryption.rst

```diff
@@ -41,11 +41,25 @@ has propagated in the cluster and Cilium knows that the IP address is an
 endpoint on a remote node, it will start encrypting packets using the encryption
 key of the remote node.
 
-The workaround for this issue is to ensure that the endpoint is not allowed to
+One workaround for this issue is to ensure that the endpoint is not allowed to
 send unencrypted traffic to arbitrary targets outside of the cluster. This can
 be achieved by defining an egress policy which either completely disallows
 traffic to ``reserved:world`` identities, or only allows egress traffic
 to addresses outside of the cluster to a certain subset of trusted IP
 addresses using ``toCIDR``, ``toCIDRSet`` and ``toFQDN`` rules.
 See :ref:`policy_examples` for more details about how to write network
 policies that restrict egress traffic to certain endpoints.
+
+Another way to mitigate this issue is to set ``encryption.strictMode.enabled``
+to ``true`` and the expected pod CIDR as ``encryption.strictMode.cidr``.
+This encryption strict mode enforces that traffic exiting a node
+to the set CIDR is always encrypted. The tradeoff being that the information
+about new pod endpoints need to propagate to the node before it can send out
+traffic to it.
+
+Enabling encryption strict mode currently has the following limitations:
+
+ - Only WireGuard encryption is supported.
+ - The pod CIDR must to be separate from the node CIDR. The easiest way to
```

Comment:
"must to be separate" sounds like should be reworded, perhaps "must be separate", or "must be different from" ?